src/template.css

```diff
@@ -0,0 +1,70 @@
+/* body{background-color: lightgray;} */
+
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+  font-family: Lato, serif;
+  font-size: 10px;
+}
+
+body {
+  width: 100%;
+  background-color: lightgray;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: large;
+}
+
+div {
+  margin-top: 10vh;
+  width: 40vw;
+  min-width: 200px;
+}
+
+h1 {
+  margin-left: auto;
+  text-align: center;
+}
+
+#taskInput {
+  width: 100%;
+  height: 30px;
+  font-size: large;
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+  height: 100%;
+}
+
+li {
+  display: flex;
```

Comment:
A good idea would be if you applied this code instead of using the property `display: flex` on each separate tag:
```
body, ul, li {
  display: flex;
}
```